packages/block-library/src/cover/style.scss

```diff
@@ -150,3 +116,42 @@
 	z-index: z-index(".wp-block-cover__video-background");
 	object-fit: cover;
 }
+
+// Styles bellow only exist to support older versions of the block.
```

Comment:
Can we expand a little bit on the differences compared to the current markup and how we're using them to target only the previous versions?

Reply:
I explained that on the description of the PR, but I agree that it should be in the comment I added it.